Sources/AardvarkMailUI/ARKEmailBugReporter.h

```diff
@@ -41,16 +41,16 @@
 @required
 
 /// Called on the main thread when a bug is filed. When not implemented, all log stores added to the bug reporter will be included.
-- (BOOL)emailBugReporter:(nonnull ARKEmailBugReporter *)emailBugReporter shouldIncludeLogStoreInBugReport:(nonnull ARKLogStore *)logStore;
+- (BOOL)emailBugReporter:(nonnull ARKEmailBugReporter *)emailBugReporter shouldIncludeLogStoreInBugReport:(nonnull ARKLogStore *)logStore NS_SWIFT_UI_ACTOR;
 
 /// Called on the main thread when a bug is filed. The attachments in the returned array will be attached to the bug report email.
-- (nullable NSArray<ARKBugReportAttachment *> *)additionalEmailAttachmentsForEmailBugReporter:(nonnull ARKEmailBugReporter *)emailBugReporter;
+- (nullable NSArray<ARKBugReportAttachment *> *)additionalEmailAttachmentsForEmailBugReporter:(nonnull ARKEmailBugReporter *)emailBugReporter NS_SWIFT_UI_ACTOR;
 
 @end
 
-typedef void (^ARKAttachmentGeneratorCompletionBlock)(ARKBugReportAttachment * _Nullable attachment);
+typedef void (^ARKAttachmentGeneratorCompletionBlock)(ARKBugReportAttachment * _Nullable attachment) NS_SWIFT_UI_ACTOR;
 
-typedef void (^ARKEmailBugReporterCustomPromptCompletionBlock)(ARKEmailBugReportConfiguration *_Nullable configuration);
+typedef void (^ARKEmailBugReporterCustomPromptCompletionBlock)(ARKEmailBugReportConfiguration *_Nullable configuration) NS_SWIFT_UI_ACTOR;
```

Comment:
Due to https://github.com/swiftlang/swift/issues/76099 these blocks will only get `@MainActor` in Xcode 16.0+

Since they're silently dropped on older Xcodes we aren't any worse off than before this change